models/payout.go

```diff
@@ -108,9 +122,17 @@ func calculatePayouts(currentTime time.Time) (map[int]float64, error) {
 
 				payouts[u.ID] += payoutPerVote
 			}
+
+			ledgerEntries = append(ledgerEntries, LedgerEntries{
```

Comment:
Shouldn't this append call be in the for loop just above, since the u.ID changes each time inside that loop?

Reply:
makes sense. Updated